### PR TITLE
Guide/anssi r45

### PR DESCRIPTION
**Pull request by a-skr**
#### Description:

- Add rules for R45 in anssi guide

#### Rationale:

- There was previously no rules attached to this recommendation.

### CODE REVIEW

File: products/debian12/profiles/anssi_bp28_enhanced.profile [hunk 1]
```diff
@@ -25,11 +25,12 @@ selections:
   - '!selinux_state'
   - '!audit_rules_mac_modification'
   - '!selinux_policytype'
-  - 'apparmor_configured'
-  - 'all_apparmor_profiles_enforced' 
-  - 'grub2_enable_apparmor'
-  - 'package_apparmor_installed'
-  - 'package_pam_apparmor_installed'
+  - '!sebool_selinuxuser_execheap'
```
Comment (Mab879) [inline products/debian12/profiles/anssi_bp28_enhanced.profile:28]: To fix the CI you will need to do the same to RHEL8-RHEL10 profiles.

**Review by Mab879 — CHANGES_REQUESTED** [review]
Thanks for the PR.

I have left some feedback to help you get the CI green.

File: products/debian12/profiles/anssi_bp28_enhanced.profile (hunk 1 above)
Comment (a-skr) [reply products/debian12/profiles/anssi_bp28_enhanced.profile:28]: I will update RHEL profiles in a few days. Thank you for your review.

**Comment by Mab879** [comment]
Waving Automatus Sanity as it is not caused by this PR.

**Review by Mab879 — APPROVED** [review]
Thanks!

**Pull request merged**